GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -165,51 +174,45 @@ protected void ReloadNodes(Func<CancellationToken, Task> loadNodesTask)
 
                 ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
+                    Nodes.Clear();
+
+                    var token = _reloadCancellationTokenSequence.Next();
+                    await loadNodesTask(token);
+
```

Comment:
After loading the nodes we should recheck if there was an cancellation request. See the original code:
```csharp
await TreeViewNode.TreeView.SwitchToMainThreadAsync(token);
```

Reply:
> After loading the nodes we should recheck if there was an cancellation request. See the original code:
> 
> ```cs
> await TreeViewNode.TreeView.SwitchToMainThreadAsync(token);
> ```

Yes, I will make that change.